tests/run-make/rustc-crates-on-stable/rmake.rs

```diff
@@ -0,0 +1,42 @@
+//! Checks if selected rustc crates can be compiled on the stable channel (or a "simulation" of it).
+//! These crates are designed to be used by downstream users.
+
+use run_make_support::{cargo, run_in_tmpdir, rustc_path, source_root};
+
+fn main() {
+    run_in_tmpdir(|| {
+        // Use the stage0 beta cargo for the compilation (it shouldn't really matter which cargo we
+        // use)
+        let cargo = cargo()
+            // Ensure `proc-macro2`'s nightly detection is disabled
+            .env("RUSTC_STAGE", "0")
+            .env("RUSTC", rustc_path())
+            // We want to disallow all nightly features to simulate a stable build
+            .env("RUSTFLAGS", "-Zallow-features=")
+            .arg("build")
+            .arg("--manifest-path")
+            .arg(source_root().join("Cargo.toml"))
+            .args(&[
+                "--config",
+                r#"workspace.exclude=["library/core"]"#,
```

Comment:
I'd also want to have a comment for this specific config, but I don't know why core needs to be excluded, why only it and not alloc or std, and if the recent library workspace is the cause or could help.

Reply:
I think that it doesn't need to be excluded anymore, as stdlib is now in a separate workspace. You can remove this (at least it works locally for me without it).

Reply:
same for me, so I've removed it.